README.md

```diff
@@ -29,66 +16,73 @@ npm i git-guppy --save-dev
 
 ## Usage
 
-### Git integration
+### 1. Declaring hooks as gulp tasks
 
-*Automatic!* 
+The first step is the most common to gulp users: you should define a new gulp
+task which name matches whichever git-hook you want to be triggerable by git. For
+instance, to define a *pre-commit* hook that runs your unit tests and then some
+custom code you could do as follows:
 
-The actual scripts that git will run to trigger guppy's hooks will be automatically
-installed to your `.git/hooks/` directory. These are just a wrapper for invoking 
-the gulp tasks that guppy registers.
+```js
+var gulp = require('gulp');
 
-Typically, a workflow can be added to your gulp tasks via a *guppy-hook*. A 
-guppy-hook is like a git-hook preconfigured for specific gulp workflows.
+gulp.task('pre-commit', ['unit'], function () {
+  // your pre-commit task's code.
+});
+```
 
-You can install *guppy-hooks* via `npm` just like any other package. For every valid
-git-hook name, there exists a "guppy-[hookname]" package that automatically installs
-the related hook to your `.git/hooks` dir, e.g. "guppy-pre-commit" or "guppy-post-update".
-Just add the guppy-hook you need as a dev-dependency in your project.
```

Comment:
Like my other comment, let's reword it so that it's clear either approach is available.